### PR TITLE
Update it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -1851,7 +1851,7 @@ msgstr "Pennello Importato"
 #: ../gui/filehandling.py:1409
 msgctxt "File→Open Recent→* confirm dialog: title"
 msgid "Open File?"
-msgstr ""
+msgstr "Apri FIle?"
```
Comment (jplloyd) [inline po/it.po]: Should the I in FIle really be capitalized?

File: po/it.po [hunk 2]
```diff
@@ -1956,12 +1956,12 @@ msgstr "Sorgente:"
...
 msgctxt "fill options: 'Source:' category (tooltip)"
 msgid "The input that the fill will be based on"
-msgstr ""
+msgstr "L'ingresso del riempimento si baserà su"
 
 #: ../gui/fill.py:584
 msgctxt "fill options: 'Source' category: Layer dropdown (tooltip)"
 msgid "Select a specific layer you want the fill to be based on"
-msgstr ""
+msgstr Seleziona un livello specifico su cui vuoi basare il riempimento"
```
Comment (jplloyd) [inline po/it.po]: Missing `"` is the cause of the build error, needs fixing.

**Comment by albanobattistella** [comment]
I'm Sorry,  "Apri File?"
________________________________
Da: Jesper Lloyd <notifications@github.com>
Inviato: lunedì 11 novembre 2019 22:44:57
A: mypaint/mypaint <mypaint@noreply.github.com>
Cc: albanobattistella <albano_battistella@hotmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [mypaint/mypaint] Update it.po (#1040)


@jplloyd commented on this pull request.

________________________________

In po/it.po<https://github.com/mypaint/mypaint/pull/1040#discussion_r344923468>:

> @@ -1851,7 +1851,7 @@ msgstr "Pennello Importato"

 #: ../gui/filehandling.py:1409

 msgctxt "File→Open Recent→* confirm dialog: title"

 msgid "Open File?"

-msgstr ""

+msgstr "Apri FIle?"



Should the I in FIle really be capitalized?

―

**Comment by jplloyd** [comment]
> I'm Sorry, "Apri File?"

Well, I would assume so but I know nothing about Italian grammar. :)

Thanks for helping out with the translations by the way. If you didn't know about it, you can also use weblate to contribute to the translation effort: https://hosted.weblate.org/projects/mypaint/mypaint/

If you amend your commit with fixes for the two issues and force push the branch, I'll merge it once the tests pass (which they should).

**Comment by albanobattistella** [comment]
I'll do It!
TX.
________________________________
Da: Jesper Lloyd <notifications@github.com>
Inviato: lunedì 11 novembre 2019 22:57:54
A: mypaint/mypaint <mypaint@noreply.github.com>
Cc: albanobattistella <albano_battistella@hotmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [mypaint/mypaint] Update it.po (#1040)


I'm Sorry, "Apri File?"

Well, I would assume so but I know nothing about Italian grammar. :)

Thanks for helping out with the translations by the way. If you didn't know about it, you can also use weblate to contribute to the translation effort: https://hosted.weblate.org/projects/mypaint/mypaint/

If you amend your commit with fixes for the two issues and force push the branch, I'll merge it once the tests pass (which they should).

**Comment by jplloyd** [comment]
Hi, just checking in case you forgot about this PR. What needs to be done is:
* in your branch, fix the two issues in `it.po`
* run `git commit -a --amend --no-edit`
* run `git push --force`

**Pull request merged**